styles.css

```diff
@@ -161,8 +177,141 @@ main .search-component .search-input {
 
 /* main-section-cards styles */
 main .title-topics-found {
-  font-size: 20px;
+  font-size: 23px;
   font-weight: 700;
   margin-top: 15px;
   margin-bottom: 20px;
 }
+
+main .cards-section {
+  width: auto;
+  height: auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+main .cards-section .card {
+  border: none;
+  border-radius: 5px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  background-color: var(--bg_default);
+  flex-basis: calc(20% - 16px);
+  min-width: 233px;
+}
+
+main .cards-section .card .topic-image {
+  height: 125px;
+  width: 100%;
+  border-radius: 5px 5px 0 0;
+  background-size: 125% auto;
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+}
+
+.topic-image.html {
+  background-image: url(./assets/html.png);
+}
+
+.topic-image.css {
+  background-image: url(./assets/css.webp);
+}
+
+.topic-image.nosql {
+  background-image: url(./assets/nosql.png);
+}
```

Comment:
it is better to move this to the HTML file as inline style.
`<div style="background-image: url('./assets/html.png')"`